x-pack/plugins/security_solution/public/flyout/left/components/host_details.tsx

```diff
@@ -126,14 +132,16 @@ export const HostDetails: React.FC<HostDetailsProps> = ({ hostName, timestamp })
         render: (user: string) => (
           <EuiText grow={false} size="xs">
             <SecurityCellActions
-              mode={CellActionsMode.HOVER_RIGHT}
-              visibleCellActions={5}
-              showActionTooltips
-              triggerId={SecurityCellActionsTrigger.DEFAULT}
               data={{
                 field: 'user.name',
                 value: user,
               }}
+              mode={CellActionsMode.HOVER_RIGHT}
+              triggerId={SecurityCellActionsTrigger.DETAILS_FLYOUT}
```

Comment:
FYI - toggle column is not working due to this [bug](https://github.com/elastic/kibana/issues/155243). Is it safer to revert back to default until it is fixed?

Reply:
I'm divided, on one side I agree we should not show an option if it doesn't work, but on the other side the old flyout shows the option... Also the day we actually fix the source of the issue we would have to make sure to put it back everywhere...
I have a feeling that @paulewing will choose to not show the option, so I'm going to prepare a commit right now to  remove it, and will merge once I have confirmation that's the behavior we want

Reply:
@christineweng see newest [commit](https://github.com/elastic/kibana/pull/164923/commits/0e97079e440042d7f6e450c3dd30cca97dedb02a)